### PR TITLE
[init] Reference flink-shade, initialize the project.

### PR DESCRIPTION
**Pull request by zhuangchong**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+            <id>apache.snapshots.https</id>
+            <name>${distMgmtSnapshotsName}</name>
+            <url>${distMgmtSnapshotsUrl}</url>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <properties>
+        <shading.prefix>org.apache.paimon.shade</shading.prefix>
+        <netty.version>4.1.82.Final</netty.version>
+        <jackson.version>2.13.4.20221013</jackson.version>
```
Comment (JingsongLi) [inline pom.xml]: netty and jackson?
We need caffeine guava and jackson

Comment (zhuangchong) [reply pom.xml]: Cleaned up, this PR is the base file for initializing the project. Soon I will add guava, caffeine, jackson modules.

**Review by JingsongLi — COMMENTED** [review]
Merged, thanks @zhuangchong for the contribution!

**Comment by JingsongLi** [comment]
Ah... I used `merge`...

**Comment by zhuangchong** [comment]
https://github.com/apache/incubator-paimon-shade/blob/1522477a1c910a4212bdc0aa0540542089b6a689/.asf.yaml#L23-L26

It is because this is the first submission, so there is no limit?

**Pull request merged**